drivers/ieee802154/Kconfig

```diff
@@ -9,11 +9,15 @@
 #
 # IEEE 802.15.4 options
 #
-menu "IEEE 802.15.4 drivers options"
+menuconfig IEEE802154
+	bool "IEEE 802.15.4 drivers options"
+	default n
+	default y if NET_L2_IEEE802154
```

Comment:
Funny, just did the same change ...

I'll merge this into the OpenThread branch ...